test/parallel/test-runner-output.mjs

```diff
@@ -219,6 +219,26 @@ const tests = [
     name: 'test-runner/output/test-runner-plan.js',
     flags: ['--test-reporter=tap'],
   },
+  {
+    name: 'test-runner/output/test-runner-force-exit.js',
```

Comment:
None of these tests actually use the force exit functionality do they?

Reply:
Not that I know of, I just didn't want to miss something with the other reporters

Reply:
#54327 is about forcing the process exit without waiting for data to be written to file. I don't think these tests as written actually verify that the issue is fixed.

Reply:
The snapshots show the output correctly written, is that accurate?

Reply:
Do these tests pass on `main` without your changes to the test runner?

Reply:
Oh oh I forgot to add the flag 🤣